lib/BC/SleighLifter.cpp

```diff
@@ -0,0 +1,1292 @@
+/*
+ * Copyright (c) 2022-present Trail of Bits, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <glog/logging.h>
+#include <remill/BC/ABI.h>
+#include <remill/BC/IntrinsicTable.h>
+#include <remill/BC/SleighLifter.h>
+#include <remill/BC/Util.h>
+
+#include <cassert>
+#include <unordered_map>
+#include <unordered_set>
+
+#include "../Arch/Sleigh/Arch.h"
```

Comment:
Use absolute path?

Reply:
Doesnt work with an internal header like this, maybe if i add it to the cmake as a private header then i could do that?